core/lib/spree/core/search.rb

```diff
@@ -0,0 +1,3 @@
+require 'spree/core/search/base'
+require 'spree/core/search/product'
+require 'spree/core/search/variant'
```

Comment:
Final newline missing.

Reply:
👍  Could you add a newline here according to the hound advice?